lib/negroku/templates/tasks/eye/eye.rb.erb

```diff
@@ -10,8 +10,8 @@ Eye.application '<%= fetch(:application)%>' do
   # except `env`, which merging down
 
   working_dir File.expand_path(File.join(File.dirname(__FILE__), %w[ processes ]))
-  stdall 'trash.log' # stdout,err logs for processes by default
-  env 'APP_ENV' => 'production' # global env for each processes
+  stdall '<%= fetch(:shared_path)%>/log/trash.log' # stdout,err logs for processes by default
+  env 'APP_ENV' => '<%= fetch(:rails_env)%>' # global env for each processes
```

Comment:
this probably came from the eye example, I don't think It have any sense here? right?